pynest/examples/gap_junctions_two_neurons.py

```diff
@@ -70,15 +70,15 @@
 
 nest.Simulate(351.)
 
-senders = nest.GetStatus(vm, 'events')[0]['senders']
-times = nest.GetStatus(vm, 'events')[0]['times']
-V = nest.GetStatus(vm, 'events')[0]['V_m']
+senders = vm.events['senders']
+times = vm.events['times']
+v_m_values = vm.events['V_m']
 
 plt.figure(1)
 plt.plot(times[numpy.where(senders == 1)],
-         V[numpy.where(senders == 1)], 'r-')
+         v_m_values[numpy.where(senders == 1)], 'r-')
```

Comment:
Can these also be on one line because of the new line length?

Reply:
here the same alignment makes it much cleaner in a single line, indeed.
Thanks for your sharp eye!

Reply:
I have been doing a lot of these changes lately so it's on my mind :smile: